clients/test/lightrider-entry-test.js

```diff
@@ -56,5 +58,18 @@ describe('lightrider-entry', () => {
       assert.strictEqual(parsedResult.runtimeError.code, LHError.UNKNOWN_ERROR);
       assert.ok(parsedResult.runtimeError.message.includes(errorMsg));
     });
+
+    it('specifies the channel as lr', async () => {
+      const runStub = sinon.stub(Runner, 'run');
```

Comment:
can you use jest mocks for this? I'm hoping we can drop `sinon` since we currently only use it in one place

Reply:
oh +1 to using jest please :)


```
const runStub = Runner.run = jest.fn()
...
const config = runStub.mock.calls[0][1].config
```

I think?

Reply:
Good point, somehow I felt I needed to use sinon in mocha tests 🙂